com.microsoft.mrtk.input/Subsystems/HandsAggregator/MRTKHandsAggregatorConfig.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// <summary>
     /// Configuration for the MRTK-specific hands aggregator.
     /// </summary>
-    public class MRTKHandsAggregatorConfig : HandsAggregatorConfig
+    public class MRTKHandsAggregatorConfig : BaseSubsystemConfig
```

Comment:
Is this intended to be in this PR?

Reply:
Yes, cleaning up the unnecessary config after a discussion with @Zee2